docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  lsp_postgres:
+    container_name: lsp_postgres
+    image: postgres:latest
+    ports:
+      - '5432:5432'
```

Comment:
maybe change the default port or specify in the readme that you need to stop the postgres service if it is already installed? otherwise there are authentication errors